### ISSUE TITLE
Trying to execute a `ddb_to_es.py` file in order to backfill OpenSearch index on my DynamoDB table (for @searchable Amplify directive)

### ISSUE THREAD

**Issue by jdanielsyeah** [aws-amplify/amplify-cli#11470]
### Amplify CLI Version

9.2.0

### Question

TLDR: I'm trying to execute a `ddb_to_es.py` file in order to backfill OpenSearch index on my DynamoDB table. But when I run the command in the terminal nothing happens.

I've made an update to my Amplify/GraphQL schema and added a @searchable directive.

I need to backfill OpenSearch index on my DynamoDB table, as per the grey info paragraph in the docs https://docs.amplify.aws/cli/graphql/search-and-result-aggregations/:

> Once the @searchable directive is added, all new records added to the model are streamed to OpenSearch. To backfill existing data, see Backfill OpenSearch index from DynamoDB table.

The docs direct to these docs: https://docs.amplify.aws/cli/graphql/troubleshooting/#backfill-opensearch-index-from-dynamodb-table

We are instructed to use the provided [python file][1] with this command:
```cli
python3 ddb_to_es.py \
  --rn 'us-west-2' \ # Use the region in which your table and OpenSearch domain reside
  --tn 'Post-XXXX-dev' \ # Table name
  --lf 'arn:aws:lambda:us-west-2:<...>:function:amplify-<...>-OpenSearchStreamingLambd-<...>' \ # Lambda function ARN, find the DynamoDB to OpenSearch streaming functions, copy entire ARN
  --esarn 'arn:aws:dynamodb:us-west-2:<...>:table/Post-<...>/stream/2019-20-03T00:00:00.350' # Event source ARN, copy the full DynamoDB table ARN
```

I've tried this with my region, ARN's, and DynamoDB references but when I hit enter in my CLI it just goes to the next command line and nothing happens? I've not used python before. There are import statements at the top of the file, but I'm only trying to run the file in isolation. Is there an environment I need to set up?

  [1]: https://raw.githubusercontent.com/aws-amplify/amplify-category-api/main/packages/graphql-elasticsearch-transformer/scripts/ddb_to_es.py
